ow_plexil/src/plexil-adapter/FaultDependencies.cpp

```diff
@@ -247,10 +247,12 @@ void FaultDependencies::parseXML(const std::string &file){
   }
   else
   {
-      ROS_INFO_STREAM("XML [" << file_name << "] parsed with errors, attr value: [" << doc.child("node").attribute("attr").value() << "]\n");
-      ROS_INFO_STREAM("Error description: " << result.description() << "\n");
-      ROS_INFO_STREAM("Error offset: " << result.offset << " (error at [..." << (file_name + result.offset) << "]\n\n");
-      return;
+      ROS_WARN_STREAM("XML [" << file_name << "] parsed with errors, attr value: [" << doc.child("node").attribute("attr").value() << "]\n");
+      ROS_WARN_STREAM("Error description: " << result.description() << "\n");
+      ROS_WARN_STREAM("Error offset: " << result.offset << " (error at [..." << (file_name + result.offset) << "]\n\n");
```

Comment:
Anything that causes a shutdown of the node should probably be logged with `ROS_ERROR_STREAM`.

Reply:
Good point, that is now changed.